content-src/asrouter/asrouter-content.jsx

```diff
@@ -333,7 +336,7 @@ export class ASRouterUISurface extends React.PureComponent {
           {this.renderOnboarding()}
           {this.renderSnippets()}
         </>,
-        this.portalContainer
+        shouldRenderInHeader ? this.headerPortal : this.footerPortal
```

Comment:
Is this going to work with having to render 2 separate things at once though? i.e. onboarding on top, then cards?

Reply:
Like don't we need  2 portals at once?

Reply:
I was thinking we would render everything from `<Trailhead/>` and the CSS would take care of making the modal vs rendering inline on top of page. What you think?

Reply:
So you would have `Trailhead` render something like:

```
<>
  <ModalOverlayWrapper>.....</ModalOverlayWrapper>
  <OnboardingCardsHere />
</>
```